test/inverse_of_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+
+class InverseOfTest < IdentityCache::TestCase
```

Comment:
Aren't there already tests for setting the inverse cached association?  If there are already tests in denormalized_has_many_test.rb, denormalized_has_one_test.rb and normalized_belongs_to_test.rb then we just need tests for this behaviour in normalized_has_many_test.rb

Reply:
No really, only this: https://github.com/Shopify/identity_cache/blob/master/test/denormalized_has_many_test.rb#L105 and I'm not sure what it tests.